plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -320,6 +319,13 @@ void GSSettingsDlg::UpdateRenderers()
 	{
 		GSRendererType ini_renderer = GSRendererType(theApp.GetConfigI("Renderer"));
 		renderer_setting = (ini_renderer == GSRendererType::Undefined) ? GSUtil::GetBestRenderer() : ini_renderer;
```

Comment:
The else branch could probably be simplified into 

```
else
	{
		renderer_setting = (GSRendererType)theApp.GetConfigI("Renderer");

		if (renderer_setting == GSRendererType::Undefined)
		{
			renderer_setting = GSUtil::GetBestRenderer();

			if (renderer_setting == GSRendererType::OGL_HW)
				SendMessage(GetDlgItem(m_hWnd, IDC_CRC_LEVEL), CB_SETCURSEL, 2, 0);
			else
				SendMessage(GetDlgItem(m_hWnd, IDC_CRC_LEVEL), CB_SETCURSEL, 3, 0);
		}
	}
```

This avoids the need of an extra local variable ``ini_renderer``